beets/importer.py

```diff
@@ -170,6 +170,22 @@ def history_get():
     return state[HISTORY_KEY]
 
 
+def calculate_quality_score(item, is_album):
+    item = get_items_as_list(is_album, item)
+    total_bitrate = sum([i.bitrate for i in item])
+    return total_bitrate / len(item)
+
+
+def get_items_as_list(is_album, item):
```

Comment:
Rather than passing `is_album` as a parameter here, you might consider using `isinstance(obj, Album)`. The function would then look like this:

```py
def items_list(obj):
    if isinstance(item, list):
        return obj
    elif isinstance(obj, Album):
        return list(obj.items())
    else:
        return [obj]
```